packages/nextjs/app/builders/0xF11046328814004583c8e74050f0d59520f53e90/page.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+interface ProfileLink {
+  name: string;
+  url: string;
+}
+
+const profileLinks: ProfileLink[] = [
+  { name: "GitHub", url: "https://github.com/Welith" },
+  { name: "Twitter", url: "https://twitter.com/bo4ka7a" }, // Replace with URL
+];
+
+const Home: React.FC = () => {
```

Comment:
Let's choose a different name + it should be NextPage (from "next") Instead of React.FC